Sources/GRPCProtobufCodeGen/ProtobufCodeGenParser.swift

```diff
@@ -166,3 +171,22 @@ extension FileDescriptor {
     return header
   }
 }
+
+extension SwiftProtobufNamer {
+  internal func trimTrailingUnderscores(from name: String) -> String {
+    var nameCopy = name
+    while nameCopy.hasSuffix("_") {
+      nameCopy = String(nameCopy.dropLast())
+    }
+    return nameCopy
+  }
+
+  internal func formattedUpperCasePackage(file: FileDescriptor) -> String {
+    let unformattedPackage = self.typePrefix(forFile: file)
+    return trimTrailingUnderscores(from: unformattedPackage)
+  }
+
+  internal func formattedLowerCasePackage(file: FileDescriptor) -> String {
+    return file.package.lowercased().replacingOccurrences(of: ".", with: "_")
```

Comment:
This assumes `file.package` is already lowercased which I don't think is necessarily the case

Reply:
Doesn't `lowercased()` handle this? Or is it not the expected behaviour?

Reply:
Sorry, I can't read 🤦‍♂️ 

Although I think we want it to be lower camel case don't we?

Reply:
Yep, we need lower camel case